internal/resources/cloud/resource_cloud_api_key.go

```diff
@@ -18,6 +18,8 @@ func ResourceAPIKey() *schema.Resource {
 	return &schema.Resource{
 		Description: `Manages a single API key on the Grafana Cloud portal (on the organization level)
 * [API documentation](https://grafana.com/docs/grafana-cloud/reference/cloud-api/#api-keys)
+
+!> Deprecated: please use ` + "`grafana_cloud_stack_service_account`" + ` and ` + "`grafana_cloud_stack_service_account_token`" + ` instead, see [Migrate API keys to Grafana service accounts using Terraform](https://grafana.com/docs/grafana-cloud/reference/migrate-api-keys-to-service-accounts/) for more information.
```

Comment:
This is the wrong resource. This is a Cloud API key, this is absolutely not deprecated. What we should have deprecated is the cloud_stack_api_key resource

Reply:
i think we have to revisit the previous PR then - https://github.com/grafana/terraform-provider-grafana/pull/872/files#diff-04961555cdd45e8547f47c42af9a867501a2b2ebf4004768ae8d8830d728f77eR28

as we deprecated the cloud_api_key and not the cloud_stack one

Reply:
see 4db6be893ebc72486661b6c6f203d9efba7fd164

Reply:
does this mean we have to make a new release no?

Reply:
Yes, I will